src/main/java/codeu/model/data/PasswordUtils.java

```diff
@@ -0,0 +1,17 @@
+package codeu.model.data;
+
+/** small class that manages password restrictions **/
+public class PasswordUtils {
+
+    public PasswordUtils() {
+    }
+
+    public boolean isPasswordCorrect(String password){
+        if(password.length()>=5&&password.length()<=13 && password.matches("(.*[a-z].*)")&& password.matches( "(.*[0-9].*)")){
```

Comment:
This is fine.. but a more "professional" way to do this would be to use java Pattern and Matcher.

See https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html. 

Regex Patterns are expensive to compile, so you would make a private static final variable for each pattern that would only be compiled once:

private static final Patter pattern CONTAINS_ALPHA = Pattern.compile("(.*[a-z].*)"); // or whatever ..

then your if statement would contain 

if (CONTAINS_ALPHA.matcher(password).matches() && CONTAINS_DIGIT.matcher(password).matches() &&.. )

Internally that is what String.matches() does, but it's inefficient because the same pattern gets re-compiled every time you run it ..